docs/overview.mdx

```diff
@@ -23,21 +23,21 @@ Casdoor serves both the web UI and the login requests from the application users
 
 1. Front-end and back-end separate architecture, developed by Golang, Casdoor supports high concurrency, provides web-based managing UI and supports multiple languages(Chinese, English).
 
-2. Casdoor supports Github, Google, QQ, WeChat third-party applications login, and support the extension of third-party login with plugins.
+2. Casdoor supports third-party applications including Github, Google, QQ, WeChat log in , and supports the extension of third-party login with plugins.
 
 3. With [Casbin](https://casbin.org/) based authorization management, Casdoor supports ACL, RBAC, ABAC, RESTful accessing control models.
 
-4. Phone verification code, email verification code and forget password features.
+4. Phone verification code, email verification code and password retrival functions.
```

Comment:
I didn't find the word: `retrival`, do you mean: `retrieval`?

Reply:
Yes, a spelling mistake, thanks for correcting!